content/reproducibility.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "The case for reproducibility"
+author: "Lindsay R. Carr"
+slug: "Reproducibility"
+date: "2016-10-04"
+description: "Explanation of reproducibility and resources to start practicing it"
+keyword1: "reproducibility"
+keyword2: "reproducible science"
+keyword3: "spreadsheets"
+output: USGSmarkdowntemplates::hugo
+---
+
+## Science is hard, why make it harder?
+
+Scientists and researchers spend a lot of time on data preparation and analysis, and some of these analyses are quite computationally intensive. The amount of time required to conduct an analysis grows for the increased complexity of calculations, amount of data, and number of datasets that will be analyzed. Many researchers use spreadsheets to conduct their analysis workflows, and apply the same set to each data set manually. A majority of the time in this type of workflow could be spent copying and pasting equations from one spreadsheet or column to another. This spreadsheet workflow is highly error-prone, as *The Economist* shows in "[Excel errors and science papers](http://www.economist.com/blogs/graphicdetail/2016/09/daily-chart-3?fsrc=scn/fb/te/bl/ed/excelerrorsandsciencepapers)".
+
+What if I told you there was a better way? 
+
+By focusing on reproducibility from the start of a project, you can quickly re-run analyses, easily share your methods with colleagues and collaborators, and apply the same methods to multiple datasets with little effort.
+
+## What does "reproducibility" really mean?
+
+The term "reproducibility" is referring to the ability for your work to be easily recreated by others. You should be able to send one or two files and a few instructions for completing your analysis. There shouldn't be a laundry list of items to change, necessary directories, or old versions of software required. 
+
+The best way to accomplish reproducibility is to start scripting your analyses. Scripting is the practice of writing code to a file in order to perform a certain task or calculation. Rather than producing "one-off" anlayses, script your work so you can reference the exact method in the future, re-run the same method for other data, and easily share your processes with colleagues, collaborators, and the public. Any scripting language will do; however, USGS Water is using R.
+
+## Tips and tricks to having reproducible workflows
+* script your work!
+* use relative (not absolute) filepaths
+* limit the number of applications/programs being used
+* keep up to date with software versions
+
+## What can you do?
+
+If you don't know where to start, try learning some basic R. There are many resources: [tryR from Code School](http://tryr.codeschool.com/), [swirl](http://swirlstats.com/), and the [USGS Introduction to R](https://owi.usgs.gov/R/training.html) Course.
+
```

Comment:
add a bullet about starting small?
Next, try to script just one piece of your analysis. Pick a set of tasks that need to be applied to several similar datasets or need to be run repeatedly. Write a script that automates one iteration through those tasks. Then reduce your analysis time and mistakes by applying that same script to all of the datasets or runs. Better still, move your code into a loop so that the script automates the repetition, too.